jans-cedarling/bindings/cedarling_python/example.py

```diff
@@ -53,8 +53,3 @@
 # show logs
 print("Logs stored in memory:")
 print(*instance.pop_logs(), sep="\n\n")
-
-example_access_token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJzb21lX3VuaXF1ZV9pZCIsImlzcyI6Imh0dHA6Oi8vZXhhbXBsZS5jb20ifQ.0EBtuIHAKtPZRKqX6LBXOZ52erMB2xuMTfVSoaYHK-o'
-request = Request(access_token=example_access_token)
-# if
-instance.authorize(request)
```

Comment:
Are we not using `authorize` as an example call?

Reply:
Yes, it is removed because it will be implemented in the 
https://github.com/JanssenProject/jans/issues/9706